README.md

```diff
@@ -4,29 +4,78 @@
 [![Build Status](https://travis-ci.org/jupyter/nbgrader.svg)](https://travis-ci.org/jupyter/nbgrader)
 [![codecov.io](http://codecov.io/github/jupyter/nbgrader/coverage.svg?branch=master)](http://codecov.io/github/jupyter/nbgrader?branch=master)
 
-A system for assigning and grading notebooks.
+A system for assigning and grading Jupyter notebooks.
 
 [Documentation can be found on Read the Docs.](http://nbgrader.readthedocs.org)
 
+
+## Visual highlights of nbgrader
+
+### Instructor toolbar extension for Jupyter notebooks
+The nbgrader toolbar extension for Jupyter notebooks guides the instructor through
+assignment and grading tasks using the familiar Jupyter notebook interface.
+
+![nbgrader toolbar extension for Jupyter notebooks](docs/source/user_guide/images/assignment_toolbar.png "nbgrader toolbar extension for Jupyter notebooks")
```

Comment:
I think this can be removed since we have a gif instead now. Markdown doesn't add borders around the images or anything so otherwise it's at first not obvious that there are two separate graphics here, so I'd just use one.